modules/packages/URL.chpl

```diff
@@ -74,25 +83,23 @@ module URL {
                 corresponding parameter of the :record:`~IO.fileReader` type.
                 Defaults to true, but when safe, setting it to false
                 can improve performance.
-  :arg start: zero-based byte offset indicating where in the file the
-              fileReader should start reading. Defaults to 0.
-  :arg end: zero-based byte offset indicating where in the file the
-            fileReader should no longer be allowed to read. Defaults
-            to a ``max(int)`` - meaning no end point.
+  :arg region: zero-based byte offset indicating where in the file the
+               fileReader should start and stop reading. Defaults to
+               ``0..`` - meaning from the start of the file to no end point.
   :returns: an open fileReader to the requested resource.
 
   :throws SystemError: Thrown if a fileReader could not be returned.
    */
   proc openUrlReader(url:string, param locking=true,
-                     start:int(64) = 0, end:int(64) = max(int(64)))
+                     region: range(?) = 0..)
                     : fileReader(locking) throws {
-    return openUrlReaderHelper(url, _iokind.dynamic, locking, start, end);
+    return openUrlReaderHelper(url, _iokind.dynamic, locking, region);
   }
 
   private
   proc openUrlReaderHelper(url:string,
                            param kind=_iokind.dynamic, param locking=true,
-                           start:int(64) = 0, end:int(64) = max(int(64)),
+                           region: range(?) = 0..,
                            style:iostyleInternal = defaultIOStyleInternal())
     : fileReader(kind, locking) throws {
```

Comment:
OK, I just saw that it has been correctly done in the first commit of this PR, but I re-introduced inadvertently `start`/`end` args when solving conflicts in commit #1ba21dd... Going to fix it!

Reply:
Yeah, that's so easy to do (I think I caused the merge conflict, too, so it's fair to blame me for letting the PR sit instead of reviewing it right away)